src/content/docs/mobile-monitoring/mobile-monitoring-ui/logs.mdx 

```diff
@@ -0,0 +1,109 @@
+---
+title: 'Mobile Logs'
+tags:
+  - Mobile monitoring
+  - Mobile monitoring UI
+  - Logs
+metaDescription: Use New Relic's Mobile Logs Monitoring UI page to view Mobile Logs
+freshnessValidatedDate: 2024-06-27
+---
+
+In mobile apps, logs can be sent to New Relic.
+
+## APIs to Capture Logs [#logs-apis]
+
+<Callout variant="important">
+### Understanding Mobile Logging delays [#mobile-logging-delays]
+
+It will take up to 15 minutes for logs to appear within New Relics Logs Tab. 
+It will take up to 15 minutes for changes to Mobile Logs toggle, sampling rate, and log level to be reflected within your mobile application.
```

Comment:
Configuration update should be within 5 mins. Is there a limitation on iOS side? Please let us know so we can consolidate.

Reply:
15 minutes is our pessimistic max time.

Reply:
We will keep it as 15 mins